pkg/sip/room.go

```diff
@@ -120,19 +122,25 @@ func (r *Room) Connect(conf *config.Config, rconf RoomConfig) error {
 					r.log.Warnw("ignoring track, room not ready", nil, "trackID", pub.SID())
 					return
 				}
-				mTrack := r.NewTrack()
-				defer mTrack.Close()
 
-				odec, err := opus.Decode(mTrack, channels)
-				if err != nil {
-					r.log.Errorw("cannot create opus decoder", err, "trackID", pub.SID())
-					return
-				}
-				defer odec.Close()
+				go func() {
```

Comment:
Isn't it running in a separate goroutine already?

Reply:
Not that I can tell from a cursory look at the GO SDK, but maybe a missed something?

Reply:
If I traced it correctly, it goes from the Pion's WebRTC (`PeerConnection.onTrack`). But I agree, lets make it explicit here.

Reply:
Right, I did not go that far back on the call stack because there is some code in the sdk part of the callback that doesn't get to run if we block here.